src/main/pages/che-7/overview/assembly_running-che-locally.adoc

```diff
@@ -50,7 +50,7 @@ $ minikube start --memory=4096
 
 === Using Minishift to set up OpenShift
 
-. Install the latest version of link:https://docs.okd.io/latest/minishift/getting-started/installing.html[Minishift] and link:https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html[set the path of the `oc` binary].
+. Install the latest version of link:https://docs.okd.io/latest/install/index.html[Minishift] and link:https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html[set the path of the `oc` binary].
```

Comment:
neither of the links work

Reply:
@rkratky , I changed the Minishift installation link and it works for me. But, the one for setting up oc binary one does not. I am looking for the correct one.

Reply:
@rkratky , strangely, the link "https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html" is now working.